src/test/java/com/puppycrawl/tools/checkstyle/checks/TranslationCheckTest.java

```diff
@@ -162,8 +164,21 @@ public void testLogOutput() throws Exception {
         final String secondErrorMessage = getCheckMessage(MSG_KEY, "anotherKey");
 
         verify(checker, propertyFiles, ImmutableMap.of(
-            getPath(""), Collections.singletonList(line + firstErrorMessage),
-            translationProps, Collections.singletonList(line + secondErrorMessage)));
+            ":0", Collections.singletonList(" " + firstErrorMessage),
+            "InputTranslationCheckFireErrors_de.properties",
+                Collections.singletonList(line + secondErrorMessage)));
+
+        verifyXml(getPath("ExpectedTranslationLog.xml"), out, (expected, actual) -> {
+            // order is not always maintained here for an unknown reason.
```

Comment:
> here for an unknown reason.

This is not good comment.
please read https://github.com/checkstyle/checkstyle/issues/5141#issuecomment-352189778